services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeYhteystietoFactory.kt

```diff
@@ -30,13 +36,24 @@ object HankeYhteystietoFactory {
     fun createDifferentiated(intValue: Int): HankeYhteystieto {
         return HankeYhteystieto(
             id = null,
-            sukunimi = "suku$intValue",
-            etunimi = "etu$intValue",
+            nimi = "etu$intValue suku$intValue",
             email = "email$intValue",
             puhelinnumero = "010$intValue$intValue$intValue$intValue$intValue$intValue$intValue",
             organisaatioId = intValue,
             organisaatioNimi = "org$intValue",
-            osasto = "osasto$intValue"
+            osasto = "osasto$intValue",
+            rooli = "Isännöitsijä$intValue",
+            tyyppi = YHTEISO,
+            alikontaktit =
+                listOf(
+                    Alikontakti(
+                        sukunimi = "suku$intValue",
+                        etunimi = "etu$intValue",
+                        email = "email$intValue",
+                        puhelinnumero =
+                            "010$intValue$intValue$intValue$intValue$intValue$intValue$intValue",
```

Comment:
Maybe extract creating the phone number from an int to a private method?

Reply:
Done.